BuffTrigger.lua

```diff
@@ -1558,4 +1580,11 @@ function BuffTrigger.GetNameAndIcon(data, triggernum)
   return name, icon;
 end
 
+function BuffTrigger.GetAdditionalProperties(data, triggernum)
```

Comment:
This function is called from the options and extends the tooltip shown at the text entries.